pyproject.toml

```diff
@@ -52,7 +52,26 @@ target-version = "py39"
 quote-style = "preserve"
 
 [tool.ruff.lint]
-ignore = ["E731", "E741"]
+select = [
+    "A",  # flake8-builtins
+    "E",  # pycodestyle
+    "ERA",  # eradicate
+    "F",  # pyflakes
+    "I",  # isort
+    "ICN",  # flake8-import-conventions
+    "ISC",  # flake8-implicit-str-concat
+    "N",  # pep8-naming
+    "PIE",  # flake8-pie
+    "T20",  # flake8-(p)print
+    "UP",  # pyupgrade
+]
+ignore = [
+    "E731",  # lambda-assignment
+    "E741",  # ambiguous-variable-name
+    "N801",  # CapWords convention
+    "N802",  # function name should be lowercase
+    "N803",  # argument name should be lowercase
```

Comment:
I think including the PEP8 naming conventions are of high importance, but I had ignore these rules because of multiple complaints from `ruff` and fixing them would pretty tiresome and added tons of changes.